src/scratchtocatrobat/converter/test_converter.py

```diff
@@ -1697,6 +1697,88 @@ def test_can_convert_answer_block(self):
+        assert isinstance(catr_brick, catbricks.CloneBrick)
+
+    # createCloneOf 
+    def test_fail_convert_create_clone_of_block_with_empty_string_arg(self):
+        sprite_name = ""
+        scratch_block = ["createCloneOf", sprite_name]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catbricks.NoteBrick)
+
+    # createCloneOf 
+    def test_fail_convert_create_clone_of_block_with_formula_arg(self):
+        scratch_block = ["createCloneOf", ["-", 2, 1]]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catbricks.NoteBrick)
+
+    # whenCloned
+    def test_can_convert_when_cloned_block(self):
+        scratch_block = ["say:", "Hello!"]
+        script = scratch.Script([30, 355, [['whenCloned'], scratch_block]])
+        catr_script = self.block_converter._catrobat_script_from(script, self.sprite_stub)
+        assert isinstance(catr_script, catbase.WhenClonedScript)
+        assert isinstance(catr_script.getBrickList().get(0), catbricks.SayBubbleBrick)
+
+    # deleteClone
+    def test_can_convert_delete_clone_block(self):
+        scratch_block = ["deleteClone"]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catbricks.DeleteThisCloneBrick)
+
+    # timeAndDate
+    def test_can_convert_time_and_date_block_second(self):
+        scratch_block = ["timeAndDate", "second"]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catformula.FormulaElement)
+        assert catr_brick.getValue() == str(catformula.Sensors.TIME_SECOND)
+
+    # timeAndDate
+    def test_can_convert_time_and_date_block_minute(self):
+        scratch_block = ["timeAndDate", "minute"]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catformula.FormulaElement)
+        assert catr_brick.getValue() == str(catformula.Sensors.TIME_MINUTE)
+
+    # timeAndDate
+    def test_can_convert_time_and_date_block_hour(self):
+        scratch_block = ["timeAndDate", "hour"]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catformula.FormulaElement)
+        assert catr_brick.getValue() == str(catformula.Sensors.TIME_HOUR)
+
+    # timeAndDate
+    def test_can_convert_time_and_date_block_weekday(self):
+        scratch_block = ["timeAndDate", "day of week"]
+        [catr_brick] = self.block_converter._catrobat_bricks_from(scratch_block, DUMMY_CATR_SPRITE)
+        assert isinstance(catr_brick, catformula.FormulaElement)
+        assert catr_brick.getValue() == str(catformula.Sensors.DATE_WEEKDAY)
```

Comment:
Concerning the off-by-one issue from above, don't forget to update this test as well.